docs/ZABBIX_PROXY_ROLE.md

```diff
@@ -269,7 +269,7 @@ These variables need to be overridden when you want to make use of the Zabbix AP
 
 * `zabbix_api_server_host`: The IP or hostname/FQDN of Zabbix server. Example: zabbix.example.com
 * `zabbix_api_use_ssl`: Is SSL required to connect to the Zabbix API server?  Default: `false`
-* `zabbix_api_server_port`: TCP port to use to connect to Zabbix server. Example: 8080
+* `zabbix_api_server_port`: 80 if `zabbix_api_use_ssl` is `false` and 443 if `true` (Default) TCP port to use to connect to Zabbix server. Example: 8080
```

Comment:
I think default for `zabbix_api_use_ssl` it `false`

Reply:
So what I was trying to say is that those two actions (combined) is the default behavior, or you can override it to whatever you want.  How would you word it?